### ISSUE TITLE
The entire codes

### ISSUE THREAD

**Issue by LuckyStar337** [hojak99/fire-detection#1]
Do you have the entire codes and coulld you open it?

I have seen the main.cpp and the codes of the key algorithms are not available.

**Comment by hojak99** [comment]
Have you applied the opencv library? this project use opencv 3.1

Can you tell me the error code?

**Comment by LuckyStar337** [comment]
Sorry, I did not express it clearly.
I have seen your codes and the vs project name is fire-smoke detection (sln file).
I want to see the way how to detect smoke (not only the fire ). So, I want to know if you have the codes about detecting smoke.

**Comment by hojak99** [comment]
In fact, I tried to add an algorithm to detect smoke later, but it was difficult. I have not implemented smoke detection, but smoke always has the property of going up. The motion of the smoke can be expressed as a vector and implemented using the motion of the vector.

**Comment by LuckyStar337** [comment]
Yes, the smoke can be detected by the motion features, but the accuracy is not high (I have tried some simple motion detection algorithm). Thanks for replying. I just want to know whether you have opened some useful smoke-detecting algorithms.
